tests/lib/rules/sort-package-json.js

```diff
@@ -54,6 +54,17 @@ new RuleTester().run('sort-package-json', rule, preprocess({
     "name": "foo",
     "version": "1.0.0"
 }`
+    },
+    // accepts and uses options
+    {
+      code: '{"version":"1.0.0","name":"foo","license":"UNLICENSED"}',
+      filename: 'package.json',
+      options: [{ sortOrder: ['license', 'name'] }],
```

Comment:
Can we have an opposite test of sortOrder: ['name', 'license']? Upstream could change the default order and this test would become useless.